gen-cpp/MakeFile.md

```diff
@@ -0,0 +1,2 @@
+all:
```

Comment:
make file usually have name Makefile.

Also we need to start using cmake: https://cmake.org/